lib/uptime_web/live/dashboard_live.ex

```diff
@@ -101,4 +72,23 @@ defmodule UptimeWeb.DashboardLive do
         {:noreply, assign(socket, services: updated_services)}
     end
   end
+
+  def handle_event("window_resize", values, socket) do
+    width = Map.get(values, "width", 800)
+    socket = assign(socket, services: list_services(calculate_checks_limit(width)))
+    {:noreply, socket}
+  end
+
+  # Leaves width of color bar ~8-9px accounting for page padding and margins between bars.
+  defp calculate_checks_limit(width) do
```

Comment:
I think this might be a good function to move to and create a file for web utils UptimeWeb.Utils, that way the services page can call it too

Reply:
ok I'll do that

Reply:
I haven't added the responsive checks limit to the service page because we still need to think about how we want to display that.